ISSUETEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## LGT GridIron Project
```

Comment:
Need an area to talk about Testing. Required `unit testing` and `e2e testing` for code changes and updates.